### DISCUSSION TITLE
How do you activate _meta.json files?

### DISCUSSION THREAD

**Discussion by cglacet** [shuding/nextra#1469] [Q&A]
I've read in the docs that you could use `_meta.json` file to structure how page should render in the nav. The customization is presented [here](https://nextra.site/docs/guide/organize-files#_metajson). But the doc doesn't state how to use that option (at first I though I only had to create such a file but apparently not). 

Here is the file structure I have for now:

```bash
❯ tree pages
pages
├── _app.tsx
├── _document.tsx
├── _meta.json
├── index.mdx
├── middleware.ts
├── posts
│   ├── index.md
│   └── pages.md
└── tags
    └── [tag].mdx

3 directories, 8 files
```

The `_meta.json` file: 

```json
{
  "index": "Accueil",
  "posts": "Billets",
  "tags": "Thèmes"
}
```

My `next.config.js` looks like this: 

```js
const withNextra = require('nextra')({
  theme: 'nextra-theme-blog',
  themeConfig: './theme.config.jsx',
  staticImage: true,
  defaultShowCopyCode: true,
  readingTime: true
})

module.exports = withNextra()
```

My `theme.config.jsx` looks like this: 

```jsx
/* https://nextra.site/docs/docs-theme/theme-configuration */
/* eslint sort-keys: error */
export default {
  darkMode: true,
  gitTimestamp({ timestamp: Date }) {
    return 'Does this have any effect? No.'
  }
}
```

On my home page I still have this showing: 

<img width="673" alt="Screenshot 2023-02-05 at 15 00 04" src="https://user-images.githubusercontent.com/28622118/216823924-f463dd76-bfee-4198-8039-59ccf3455634.png">

Where "About" and "Posts" are the markdown meta `title: ...` respectively defined in `index.md` and `posts/index.md`.

**Comment by ezbeazy** [comment]
Can we get _meta.json to work with blog theme pages? Or hide pages with md frontmatter?

**Comment by hariria** [comment]
Yes you can use the `_meta.ts` file directly under the `pages` directory to structure their layout in the navbar. This is an example from the alpha release of Nextra v3: See https://github.com/shuding/nextra/blob/v3/examples/swr-site/pages/en/_meta.ts for more info.

**Reply by kidfribble** [reply]
Nice! This seems like the most powerful option, but unfortunately (at least to me) it looks like we're locked into the basic json ordering method in stable versions.

I'm personally just trying to show/hide options from here based on user auth. Gonna test the alpha version and see if it doesn't break everything else :)
